APPS-1428 Update ofh azure region

Gvaihir opened the pull request:
(feat) Developing.md - updated docs
(feat) elsewhere - added new region

Gvaihir committed:
APPS-1428 ofh azure
(feat) Developing.md - updated docs
(feat) elsewhere - added new region

Gvaihir committed:
APPS-1428 ofh azure
(fix) Developing.md - minor edits

Gvaihir committed:
APPS-1428 ofh azure
(fix) RELEASE_NOTES.md - update

commandlinegirl commented on scripts/dxcompiler_copy/dxapp.json:
```diff
@@ -99,10 +99,17 @@
   },
   "details": {
-    "whatsNew": "* 0.0.2: Added new London region."
+    "whatsNew": "* 0.0.3: Added new London OFH region * 0.0.2: Added new London region."
```
```suggestion
    "whatsNew": "* 0.0.3: Added new London OFH region\n* 0.0.2: Added new London region."
```

Gvaihir replied:
fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)

commandlinegirl commented on scripts/multi_region_tests.py:
```diff
@@ -25,7 +25,8 @@
             "dxCompiler_Amsterdam",
-            "dxCompiler_Berlin"]
+            "dxCompiler_Berlin",
+            "dxCompiler_OFH_TRE"]
```
Hmm.. It looks like dxCompiler_London and dxCompiler_Europe_London are missing from here.

Gvaihir replied:
yes, I mentioned that in the comments to APPS-1428 - was not sure if that's intended or just an oversight. I'll add them

Gvaihir replied:
fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)

commandlinegirl commented on scripts/build_release.py:
```diff
@@ -37,7 +37,8 @@
     "aws:eu-west-2": "dxCompiler_London",
-    "aws:eu-west-2-g": "dxCompiler_Europe_London"
+    "aws:eu-west-2-g": "dxCompiler_Europe_London",
+    "azure:uksouth-ofh": "dxCompiler_OFH_TRE"
```
How about we name it dxCompiler_OFH_TRE_London (in case there will be a new region in the future for OFH-TRE)?

Gvaihir replied:
Sure

Gvaihir replied:
fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)

commandlinegirl reviewed (changes requested):
Mostly minor requests. Thank you!

Gvaihir committed:
Merge branch 'develop' into APPS-1428_ofh_azure

# Conflicts:
#	RELEASE_NOTES.md

Gvaihir committed:
APPS-1428 ofh azure
(fix) multi_region_tests.py - added other London regions which were not included previously.
(fix) elsewhere - minor fixes and renaming

Gvaihir committed:
APPS-1491 dxda version bump
(fix) dxCompiler_project_ids_* - removed files, content is in Confluence

commandlinegirl reviewed (approved):
LGTM, thank you!
